build.properties

```diff
@@ -10,10 +9,12 @@ mod_url=https://github.com/Team-RTG/Realistic-Terrain-Generation
 mod_author="Team RTG"
 mod_creds=Forked from RWG by ted80. OpenSimplex Noise by KdotJPG.
 mod_logo=assets/rtg/logo.png
-mcf_version=12.18.1.2011
-# mcf_postfix is the branch postfix (without '-') on the Forge version when it's not the default branch in the Forge repo
-# This should be same as mc_version, eg Non-default: 1.9.4-12.17.0.1974-1.9.4, Default: 1.9.4-12.17.0.1976
-mcf_postfix=
-mappings=snapshot_nodoc_20160724
+mcf_version=12.18.1.2094
```

Comment:
The thing is, most users are going to be using the `latest` Forge, or atleast something newer than the `recommaneded`. It's best if we build against the `latest` to ensure compatibility with what users are using. If there are any incompatibilities we'll likely catch them in development, or initial testing. If we build against `recommended` and most users are using `latest` and run into issues, it's going to be a headache to track something down that is related to a Forge update.

Also, the Forge devs are quite anal about introducing changes that would break compatibility. If something was introduced that would break old versions there would likely be warnings issued. Things that could possibly break compatibility are usually reserved for major updates (ie. waiting until 1.11 before enforcing lowercase-only mod IDs)

After watching Forge for a while now, it seems the 'recomended' version is set more by guildline than by rule. In the past there's been versions set to 'recommended' directly after code has been committed and not really tested (publically), and other times, when the recommended version should be bumped, it isn't. (ie The recommended version for 1.7.10 is still 1566, but could have been bumped to 1614 a year ago, since it's the version almost everyone uses and has been thoroughly tested by now.)

#### As for those commits:
> Fix async race condition on new chunks

This shouldn't affect anything in RTG. The only thing it would have affected is the village code that we've already removed, but this would have fixed things, rather than break them. (No more broken buildings in villages, and possibly no more random undecorated chunks.)

> Update decorate for biomes to support Decorate event

This updates biome decoration events to add support for 1.10 added stuff, which shouldn't affect anything we're doing, though in the future we should update to add support for fossils, etc. This should be transparent to RTG.

> Fix a vanilla bug with the blockstate ID map not being properly symmetric with respect to block.getStateFromMeta

This seems irrelevant, since `Block#getStateFromMeta` is deprecated and just returns `Block#getDefaultState` regardless of the meta value passed. Everything has moved to properties. I'm not sure the purpose why this was even commited to begin with, though when this patch was initially being written it was for MC 1.8. This shouldn't have any direct affect on RTG.

> Fix vanilla bug in BlockFalling that caused blocks to loose state info during world gen

This is an OLD vanilla bug. It shouldn't affect anything that we do, but vanilla blocks that can fall shouldn't remain suspended anymore with no solid blocks below them. This should be completely transparent to RTG.

------
If you're still worried about it, you can push the version back, or we can do another snapshot after building against the `latest` Forge and wait another week until release. I know the plan was to release this weekend, and there are users looking for a stable RTG release, but that shouldn't be a reason to rush it.